config.json

```diff
@@ -24,6 +24,12 @@
       "public_url": "/"
     },
 
+    "next": {
+      "domain": "next.kbase.us",
+      "legacy": "legacy.next.kbase.us",
+      "public_url": "/"
+    },
+
```

Comment:
This is the only place that was required to be updated to prevent the 404s ran into. It is the situation "using a deployed build with an undefined environment" which usually does not happen during development since the defaults from the `.env` file are used. We might want to update `scripts/build_deploy.sh` to use defaults from `.env`.